cinema/serializers.py

```diff
@@ -2,0 +1,103 @@
+from rest_framework import serializers
+
+from cinema.models import (
+    CinemaHall,
+    Genre,
+    Actor,
+    Movie,
+    MovieSession
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    capacity = serializers.IntegerField(read_only=True)
+
+    class Meta:
+        model = CinemaHall
+        fields = (
+            "id",
+            "name",
+            "rows",
+            "seats_in_row",
+            "capacity"
+        )
+
+    def get_capacity(self, obj):
+        return obj.rows * obj.seats_in_row
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = (
+            "id",
+            "name"
+        )
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = (
+            "id",
+            "first_name",
+            "last_name",
+            "full_name"
```

Comment:
The `full_name` field in `ActorSerializer` should be defined as a read-only field since it is a property and not a model field. You can achieve this by using `serializers.ReadOnlyField()`.